lib/plugin.js

```diff
@@ -130,3 +129,18 @@ function googleOptimize({ experiment }) {
 
   window.ga('set', 'exp', exp)
 }
+
+// should we skip bots?
+function skipAssignment(ctx) {  
+  if(!<%= options.excludeBots %>) return false
```

Comment:
Neat: space before `(` and please use `{ }` for consistency

Reply:
Space - I'm surprised the aggressive eslint rules didn't catch this.
IMHO all bool functions should always return bool, not something that's falsy, but I can change it.

Reply:
Indeed long time issue with lodash templates that are not lintable. There is [eslint-multiplexer](https://github.com/pimlie/eslint-multiplexer) ([usage](https://github.com/nuxt/nuxt.js/blob/dev/package.json#L23)) but not easy to use. Working on a more linter friendly template engine.